content/blog/2023/czi-mentorship-round-2.md

```diff
@@ -0,0 +1,135 @@
+title=Nextflow and nf-core Mentorship, Round 2
+date=2023-04-17
+type=post
+description=With the second round of the mentorship program now complete, we celebrate the success of the most recent cohort of mentors and mentees.
+image=img/mentorships-round2-social.jpg
+tags=nextflow,nf-core,czi,mentorship,training
+status=published
+author=Chris Hakkaart
+icon=chris_h.png
+~~~~~~
+
+## Introduction
+
+<div class="pull-right" style=" max-width: 350px; margin-left: 2rem; padding: 2rem;">
```

Comment:
It's a lovely image, but I find the size a bit overwhelming.. This seems ok for me:

```suggestion
<div class="pull-right" style=" max-width: 30%; margin-left: 2rem; padding: 2rem;">
```